unfoldingWord/translationCore
# Created check data loading actions

#### This pull request addresses:

**Loading Data from Data Persistence #985**
Added loading actions to load comments, reminders, selections and verseEdits data from the filesystem


#### How to test this pull request:
- Import the loading actions in ToolsContainer.js and then call them inside a componentWillMount() function.
- Make sure you have the files previously saved in the file system in the appropriate directory.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1006)
